README.md

```diff
@@ -109,11 +109,11 @@ then run the following commands:
 
     git clone https://github.com/LNP-BP/rgb-node.git
     cd rgb-node
-    cargo build --release --bins
+    cargo install --locked --root . rgb_node --features all
```

Comment:
Well, this does not compile the local code from the repo, but pulls the latest release from the crates.io. Is this the intentend change? If yes, then the rest above is not needed (cloning git repo etc).

Reply:
Sorry, I didn't see it.